runtime/ts/storage/crdt-collection-model.ts

```diff
@@ -85,15 +90,15 @@ export class CrdtCollectionModel {
   }
 
   // [{id, value, keys: []}]
-  toLiteral(): {id, value, keys}[] {
-    const result: {id, value, keys}[] = [];
+  toLiteral(): Model[] {
```

Comment:
This change makes me realize that 'Model' isn't a good name for the interface. We don't want to have a `CrdtCollectionModel`'s `Model`, that's confusing. Even more confusing is that this interface represents a single `Entity`, not all of the entities in the model.

How about `Entity` or `EntityData`?